ludwig/benchmarking/utils.py

```diff
@@ -155,12 +159,80 @@ async def download_one(
     return dataset_name, local_dir
 
 
+def validate_benchmarking_config(benchmarking_config: Dict[str, Any]) -> None:
```

Comment:
nit: Consider adding a simple unit test for this function. And/or a test that validates the benchmarking config that's currently checked in.

Reply:
Done